.storybook/stories/button/button-group.stories.ts

```diff
@@ -26,7 +26,7 @@ export default {
   component: ClrButtonGroup,
   argTypes: {
     // inputs
-    clrMenuPosition: { control: { type: 'radio', options: CLR_MENU_POSITIONS } },
+    clrMenuPosition: { type: 'radio', options: CLR_MENU_POSITIONS },
```

Comment:
```suggestion
    clrMenuPosition: { control: 'radio', options: CLR_MENU_POSITIONS },
```

Reply:
Please fix all of these. Replace `type: 'radio'` with `control: 'radio'`.

Reply:
Fixed the above.